tests/async_tests/test_interfaces.py

```diff
@@ -223,8 +223,8 @@ async def test_proxy_https_requests(
     https_server: Server,
 ) -> None:
     method = b"GET"
-    url = (b"https", *https_server.netloc, b"/")
-    headers = [https_server.host_header]
+    url = (b"https", b"example.org", 443, b"/")
+    headers = [(b"host", b"example.org")]
```

Comment:
Switching these back to using `example.org`, since configuring certs for going through mitm-proxy was quite iffy.

Reply:
https://github.com/encode/httpcore/pull/199 has been merged, so probably it can be changed back?